packages/aragon-wrapper/src/index.js

```diff
@@ -849,8 +849,12 @@ export default class Aragon {
         // Also, at the same time it's a hack for checking if the call will revert,
         // since `eth_call` returns `0x` if the call fails and if the call returns nothing.
         // So yeah...
-        await this.web3.eth.estimateGas({ ...directTransaction })
+        const estimatedGasLimit = await this.web3.eth.estimateGas({ ...directTransaction })
+        const recommendedGasLimit = await getRecommendedGasLimit({ web3: this.web3, estimatedGasLimit })
 
+        if (directTransaction.gas && directTransaction.gas < recommendedGasLimit) {
```

Comment:
If no `gas` is given we just fall back to MetaMask's recommended gas limit calculation which happens in the extension.  The calculation is identical except for the block gas limit upper hard cap (which is 95% as opposed to 90%)

Reply:
I would say we should always calculate the gas limit of the transaction even if it is not provided, this way we can use the the `getDefaultGasPrice` function from #158.

This will also remove the need for having to manually calculate the gas limit sending the transactions that aragon.js returns if not using Metamask (for example when using the `dao exec` command in the CLI, which uses aragon.js to generate the transaction and then needs to set the gas limit before sending it).